packages/clerk-js/src/core/resources/OrganizationMembership.ts

```diff
@@ -34,30 +34,16 @@ export class OrganizationMembership extends BaseResource implements Organization
       // `paginated` is used in some legacy endpoints to support clerk paginated responses
       // The parameter will be dropped in FAPI v2
       search: convertPageToOffset({ ...retrieveMembershipsParams, paginated: true }),
-    })
-      .then(res => {
-        if (!res?.response) {
-          return {
-            total_count: 0,
-            data: [],
-          };
-        }
-
-        // TODO: Fix typing
-        const { data: suggestions, total_count } =
-          res.response as unknown as ClerkPaginatedResponse<OrganizationMembershipJSON>;
-
-        return {
-          total_count,
-          data: suggestions.map(suggestion => new OrganizationMembership(suggestion)),
-        };
-      })
-      .catch(() => {
-        return {
-          total_count: 0,
-          data: [],
-        };
-      });
+    }).then(res => {
+      // TODO: Fix typing
```

Comment:
❓ is this different from the above `as unknown as ClerkPaginatedResponse...` ? I think we should remove this TODO or add also to the other refs of this PR.
Let's also add a ticket to make this refactoring part of the v6 project

Reply:
I believe we already have one. It is a general issue with `BaseResource._fetch` and type inference.